viewer/app/index.html

```diff
@@ -16,11 +16,11 @@
   <link rel="stylesheet" href="styles/bundled.css">
   <link rel="canonical" href="https://googlechrome.github.io/lighthouse/viewer/">
 </head>
-<body class="lh-root lh-vars">
+<body>
 
-<div class="drop_zone"></div>
+<div class="renderer-container"></div>
```

Comment:
FWIW my suggestion was to keep this as a `main` tag and have generated report contents as `div`s. Generating `main` in `renderReport` might create nested `main`s in the flow report.